config/transformers/shadowToCss.test.ts

```diff
@@ -0,0 +1,171 @@
+import {getMockToken} from '~/src/test-utilities'
+import {shadowToCss} from './shadowToCss'
+
+describe('Transformer: shadowToCss', () => {
+  it('transforms `shadow` token to css shadow string', () => {
+    const input = [
+      getMockToken({
+        value: {
+          color: '#000000',
+          offsetX: '0px',
+          offsetY: '2px',
+          blur: '1px',
+          spread: '0'
+        }
+      })
+    ]
+    const expectedOutput = ['0px 2px 1px 0 #000000']
+    expect(input.map(item => shadowToCss.transformer(item))).toStrictEqual(expectedOutput)
+  })
+
+  it('transforms inset `shadow` token to css shadow string', () => {
+    const input = [
+      getMockToken({
+        value: {
+          color: '#000000',
+          offsetX: '0px',
+          offsetY: '2px',
+          blur: '1px',
+          spread: '0px',
+          inset: true
+        }
+      }),
+      getMockToken({
+        value: {
+          color: '#000000',
+          offsetX: '0px',
+          offsetY: '2px',
+          blur: '1px',
+          spread: '0px',
+          false: true
```

Comment:
should this be `inset`?

```suggestion
          inset: true
```